doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,326 @@
-other artifacts.
+.. note::
+    This guide is relevant for all DataParallelTrainer, including TorchTrainer and TensorflowTrainer.
 
-Ray Train also exposes logging callbacks that automate some of these tasks.
+Most experiment tracking libraries work out-of-the-box with Ray Train. 
+This guide provides instructions on how to set up the code so that your favorite experiment tracking libraries 
+can work for distributed data parallel training with Ray Train. We will conclude the session with debugging
+tips.
 
-.. _train-experiment-tracking-native:
+Before we begin, the following is roughly how you can use the native experiment tracking lirary calls 
+inside of Ray Train. 
 
-Using native experiment tracking libraries
-------------------------------------------
+.. code-block:: python
 
-.. tab-set::
+    from ray.train.torch import TorchTrainer
+    from ray.train import ScalingConfig
 
-    .. tab-item:: PyTorch
+    def train_func(config):
+        # Training code and native lexperiment tracking library calls go here.
 
-        You can use experiment tracking libraries such as Weights & Biases, Mlflow, or
-        Comet directly in your Ray Train training loop.
+    scaling_config = ScalingConfig(num_workers=2, use_gpu=True)
+    trainer = TorchTrainer(train_func, scaling_config=scaling_config)
+    result = trainer.fit()
 
-        There are two things to keep in mind:
+Ray Train lets you use native experiment tracking libraries by customizing the tracking 
+logic inside the ``train_func`` function. In this way, you can port your experiment tracking 
+logic to Ray Train with minimal changes. 
 
-        1. Your code is executed in parallel on many workers. However, you often only want to report
-        results from one of these workers (usually the first worker - the "rank 0" worker).
+Log distributed training experiments
+====================================
 
-        2. When using the native libraries, you should report the results to both Ray Train and
-        the experiment tracking library.
+For some scenarios, every Worker generates an identical copy and saving a single copy is sufficient. 
+Ray Train lets you apply logic to only the rank 0 Worker with the following method:
+:meth:`context.get_world_rank() <ray.train.context.TrainContext.get_world_rank>`.
 
-        Example:
+There are roughly two kinds of experiment tracking modes: online and offline. For online mode, you log
+towards a tracking service that is running. Usually you need credentials to access the service.
+For offline mode, you log towards local file directory. Usually no credentials are needed.
 
-        .. code-block:: python
+For online mode, it is important to make sure that all nodes and Worker processes have 
+access to credentials. For offline mode, major consideration is to ensure that there is a 
+shared file system where all nodes can write to.
+
+Step 1: Choose a tracking service to use
+----------------------------------------
```

Comment:
I think the five steps are too involved for the simple message that they convey.

I think we can simplify this with a short list:

1. Initialize the tracking service
2. Log results to the service
3. Clean up the session

and then use the short tabbed examples that we already have below to illustrate that

Reply:
ah yes, updated this section completely:

- Set up to connect to a tracking backend
- Configure and launch a run
- Log
- Finish the run